src/lib/containers/ErrorBanner.tsx

```diff
@@ -85,3 +75,29 @@ export const ErrorBanner = (props: ErrorProps) => {
     </div>
   )
 }
+
+export const ErrorFallbackComponent: React.FunctionComponent<FallbackProps> = ({
+  error,
+  resetErrorBoundary,
+}) => {
+  return (
+    <div role="alert" className="bootstrap-4-backport SRC-marginBottomTop">
+      <ErrorBanner error={error}></ErrorBanner>
+      <Button onClick={resetErrorBoundary}>Reload</Button>
+    </div>
+  )
+}
+
+/**
+ * ErrorBoundary component that uses the default error fallback component, unless overridden.
+ * Internally uses `react-error-boundary`.
+ *
+ * Use with {@link react-error-boundary#handleError | handleError}
+ * @param props
+ * @returns
+ */
+export const SynapseErrorBoundary: React.FC<
+  Optional<ErrorBoundaryPropsWithComponent, 'FallbackComponent'>
+> = (props: Optional<ErrorBoundaryPropsWithComponent, 'FallbackComponent'>) => (
+  <ErrorBoundary FallbackComponent={ErrorFallbackComponent} {...props} />
+)
```

Comment:
Now, use this instead of `ErrorBoundary` because our default fallback component is baked-in